skimage/util/process.py

```diff
@@ -0,0 +1,33 @@
+import dask.array as da
+
+__all__ = ['process_chunks']
+
+
+def process_chunks(function, array, chunks=None, depth=None, mode=None):
```

Comment:
Agree that we could improve upon `process`.  

This could also probably use a docstring saying that it splits apart an array and executes a normal function in parallel on overlapping chunks of the array.